uwblueprint/planet-read
# Style homepage according to visual explorations

## Notion ticket link

[Convert Homepage to Chakra UI](https://www.notion.so/uwblueprintexecs/a658c933d18c48d4b43fd99bda2bd3b4?v=4a47edc475b34bb99b25ff1fae2a5507&p=606dc4347e584cc492e71ac666772288)



## Implementation description
* modified design system colors 
* created ButtonRadioGroup 
* styled primary & tertiary buttons
* used ButtonRadioGroup for level selection and translator/review toggle aswell as my work/browse stories toggle
  * deleted ToggleButton 
*  styled Filter for highfi
* styled StoryCard  for highfi
  * styled Badge & Headings for highfi
* migrated Header to chakra ui and styled for highfi
* styled Homepage for highfi


## Steps to test
1. log in 
2. poke around
![image](https://user-images.githubusercontent.com/15113249/130904115-4f96b0bc-cad5-4189-bcd4-f3b6e5c99210.png)
observe that the filter and header are not sticky
![image](https://user-images.githubusercontent.com/15113249/130904157-dffe5067-4c30-4bfe-b14c-2c683156f355.png)


## What should reviewers focus on?
* does the ui feel useable 
* is the code readable
* is the code reusable (in particular, does the ButtonRadioGroup & it's usage make sense?)


## Checklist
